v2/cli/init_templates.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/Masterminds/sprig"
+)
+
+// Template project.yaml file for a non-git project
+// nolint: lll
+var projectTemplate = []byte(`# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
+apiVersion: brigade.sh/v2-alpha.5
+kind: Project
+metadata:
+  id: {{ .ProjectID }}
+description: My new Brigade project
+spec:
+  eventSubscriptions:
+  - source: brigade.sh/cli
+    types:
+    - exec
+  workerTemplate:
+    # logLevel: DEBUG
+    {{- if eq .GitCloneURL "" }}
+    defaultConfigFiles:
+      brigade.{{ .Language }}: |
+        {{ nindent 8 .Script }}
+    {{- else }}
+    git:
+      cloneURL: {{ .GitCloneURL }}
+    {{- end }}
+`)
+
+// nolint: lll
+var scriptTemplate = []byte(`{{- if eq .Language "js" }}const { events } = require("@brigadecore/brigadier");
```

Comment:
Can this be broken into two lines? That would be more readable.

But I know templates are weird. There could be reasons...

Reply:
You can also try a `-` before the closing curlies to see if that helps.

Reply:
That fixed it, thanks!